engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -641,6 +644,10 @@ void printUsage(String cmd)
         null,
         null,
         "Exit the CLI"));
+    globalCommands.put("quit", new CommandSpec(new ExitCommand(),
```

Comment:
Similar to my answer in https://issues.apache.org/jira/browse/APEXCORE-559 , I would say most command line tools I have used support "quit" and I think a lot of users would be surprised to find it missing in the Apex Cli. Bash is a Unix shell and I think they want to limit the reserved words as shell commands.